Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -627,9 +627,9 @@ void MediaSource::streamEndedWithError(std::optional<EndOfStreamError> error)
         setDurationInternal(maxEndTime);
 
         // 2. Notify the media element that it now has all of the media data.
+        m_private->markEndOfStream(MediaSourcePrivate::EosNoError);
```

Comment:
Can you add a comment in the change log telling why we should do this particular change?

Reply:
ended up removing that part, it's what caused GTK errors.